README.md

```diff
@@ -64,12 +88,42 @@ Cloning into 'NetSoakTest'...
 > cmake -S C:/Path/To/o3de -b C:/Path/To/o3de/build -G "Visual Studio 16 2019" -DLY_3RDPARTY_PATH="C:/3rdparty" -DLY_PROJECTS=NetSoakTest 
 
 # example build command
-> cmake --build C:/Path/To/o3de/build --target Editor NetSoakTest.GameLauncher --configure profile -- /m /nologo 
+> cmake --build C:/Path/To/o3de/build --target Editor NetSoakTest.ServerLauncher --configure profile -- /m /nologo 
 
 ```
 
 If you have a Git credential helper configured, you should not be prompted for your credentials anymore.
 
+## Running the Project
+
+Run the netsoak ServerLauncher with the relevant options (see below). It is strongly recommended to use --rhi=null when launching NetSoakTest.ServerLauncher
```

Comment:
I wish there were docs about what the RHI parameter does so we could link to it here, but a search on o3de.org comes up empty. Might just mention that will disable rendering in order to make a headless server.